app/api/leaderboard/functions.ts

```diff
@@ -27,6 +27,7 @@ type OrderingKey = LeaderboardSortKey | `-${LeaderboardSortKey}`;
 export const getLeaderboardData = async (
   dateRange: readonly [Date, Date],
   ordering: OrderingKey,
+  role: string,
```

Comment:
```suggestion
  role: "any" | "core" | "intern" | "contributor",
```

Reply:
will change after implementing multi select , since it requires type as array or something like eg: core,intern